book/website/community-handbook/onboarding-offboarding/onboarding-core-team.md

```diff
@@ -0,0 +1,64 @@
+# ☀️ Welcome to *The Turing Way* Core Team!☀️
+
+:::success
+This onboarding document is a living document – as is *The Turing Way* itself. As you'll learn from being in this community: this is a constantly-evolving project that holds [emergence as a core value](https://hackmd.io/kVkwKSYuSLWWhLzlNSuT0g). In other words: as the project evolves – so will the team that enables its sustainability as a project.
+
+From April to August 2022, Anne (TTW CM, 2022) documented [documented her research of/within *The Turing Way* community on Github](https://github.com/alan-turing-institute/the-turing-way/discussions/2552). Much of that research has been used to make this onboarding document.
+:::
+
+[TOC]
+
+# TTW Core Team Onboarding checklists
+- [ ] Review [TTW Onboarding document](https://hackmd.io/pBysnLn2QxKMCHqPKvj9uA?both) (this document!)
+- [ ] Review [TTW Offboarding document](https://hackmd.io/@turingway/ttw-core-team-offboarding) (not to be used for now!)
+
+## Accounts and Permissions
+- [ ] Add to #ttw-core-team channel on Slack
```

Comment:
As the core team is transitioning to Working groups I would add the link to the WG for newcomers to find common interests.

Reply:
Added in qualifier at the top that these are primarily being archived here. Once we have the WG PR up (#2729 #3287), can add it back here.